File: 1/main.py
from typing import List


class Solution:
    def validStrings(self, n : int, k : int, arr : List[str]) -> int:
        ans=0
        for el in arr:
            C=0
            for e in el:
                if e in "aeiou":
                    C+=1
            if C==k:
                ans+=1
        return ans

File: 1/main.java
public class Solution 
{
    public static int validStrings(int n, int k, String[] arr) 
    {
        int totalValid = 0;

        for (String str : arr) {
            int vowelCount = 0;
            for (char ch : str.toCharArray()) {
                if ("aeiou".indexOf(ch) >= 0) {
                    vowelCount++;
                }
            }
            if (vowelCount == k) {
                totalValid++;
            }
        }

        return totalValid;
    }
}
